panel/dist/css/chat_entry.css

```diff
@@ -0,0 +1,72 @@
+.chat-entry {
+  display: flex;
+  flex-direction: row;
+}
+
+.avatar {
```

Comment:
I'm starting to think that we should put the `chat` components in their own namespace. I.e. in `pn.chat` instead of `pn.widgets`. What do you think?

Reply:
I'm wondering whether there is some rule of thumb for this?